java/pom.xml

```diff
@@ -111,8 +112,30 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.6.1</version>
           <configuration>
-            <source>1.7</source>
-            <target>1.7</target>
+          <source>8</source>
+          <target>8</target>
+          <encoding>UTF-8</encoding>
+              <compilerArgs combine.children="append">
```

Comment:
Is this the only POM that needs updating? There's many, not sure how this works.

Reply:
I think this would be the only one. But evidently Mike and Deanna have bigger better plans, so they're going to learn from this but do something better.